### PR TITLE
[MODULAR] adjusts some modular balloon alerts to fit tg-guideline

### PR DESCRIPTION
**Pull request by desertmote**
## About The Pull Request

i had vsc open so why not

## How This Contributes To The Skyrat Roleplay Experience

blablabla modular content should still follow upstream implementation guidlines blababla

## Changelog
N/a

### CODE REVIEW

File: modular_skyrat/modules/customization/game/objects/items/devices/ttsdevice.dm [hunk 1]
```diff
@@ -9,13 +9,13 @@
 	obj_flags = UNIQUE_RENAME
 
 /obj/item/ttsdevice/attack_self(mob/user)
-	user.balloon_alert_to_viewers("Typing on [initial(name)].", "You begin typing.")
+	user.balloon_alert_to_viewers("typing on [initial(name)]...", "you begin typing...")
```
Comment (tf-4) [inline modular_skyrat/modules/customization/game/objects/items/devices/ttsdevice.dm]:
```suggestion
	user.balloon_alert_to_viewers("typing on [initial(name)]...", "started typing...")
```
Could make it slightly more concise here

File: modular_skyrat/modules/customization/game/objects/items/devices/ttsdevice.dm [hunk 2]
```diff
@@ -9,13 +9,13 @@
...
 	playsound(src, 'modular_skyrat/master_files/sound/items/tts/started_type.ogg', 50, TRUE)
 	var/str = tgui_input_text(user, "What would you like the device to say?", "Say Text", "", MAX_MESSAGE_LEN, encode = FALSE)
 	if(QDELETED(src) || !user.canUseTopic(src, BE_CLOSE))
 		return
 	if(!str)
-		user.balloon_alert_to_viewers("Stops typing on [initial(name)].", "You stop typing.")
+		user.balloon_alert_to_viewers("stops typing on [initial(name)]", "you stop typing")
```
Comment (tf-4) [inline modular_skyrat/modules/customization/game/objects/items/devices/ttsdevice.dm]:
```suggestion
		user.balloon_alert_to_viewers("stops typing on [initial(name)]", "stopped typing")
```
And here

**Comment by desertmote** [comment]
even better!

File: modular_skyrat/modules/customization/game/objects/items/devices/ttsdevice.dm [hunk 3]
```diff
@@ -9,13 +9,13 @@
 	obj_flags = UNIQUE_RENAME
 
 /obj/item/ttsdevice/attack_self(mob/user)
-	user.balloon_alert_to_viewers("Typing on [initial(name)].", "You begin typing.")
+	user.balloon_alert_to_viewers("typing on [initial(name)]...", "started typing...")
```
Comment (tf-4) [inline modular_skyrat/modules/customization/game/objects/items/devices/ttsdevice.dm]:
```suggestion
	user.balloon_alert_to_viewers("starts typing...", "started typing...")
```
Actually I'm not sure the name of the tts device is very good to have in a balloon alert

File: modular_skyrat/modules/customization/game/objects/items/devices/ttsdevice.dm [hunk 4]
```diff
@@ -9,13 +9,13 @@
...
 	playsound(src, 'modular_skyrat/master_files/sound/items/tts/started_type.ogg', 50, TRUE)
 	var/str = tgui_input_text(user, "What would you like the device to say?", "Say Text", "", MAX_MESSAGE_LEN, encode = FALSE)
 	if(QDELETED(src) || !user.canUseTopic(src, BE_CLOSE))
 		return
 	if(!str)
-		user.balloon_alert_to_viewers("Stops typing on [initial(name)].", "You stop typing.")
+		user.balloon_alert_to_viewers("stops typing on [initial(name)]", "stopped typing")
```
Comment (tf-4) [inline modular_skyrat/modules/customization/game/objects/items/devices/ttsdevice.dm]:
```suggestion
		user.balloon_alert_to_viewers("stops typing", "stopped typing")
```

**Review by tf-4 — COMMENTED** [review]
Not sure how the proc works and if you need two args, on phone so can't really check but see if you can do it with one

**Pull request merged**